fairroot/basemq/devices/FairMQUnpacker.h

```diff
@@ -105,7 +105,7 @@ class FairMQUnpacker : public fair::mq::Device
 
     void Run()
     {
-        fair::mq::Channel& inputChannel = fChannels.at(fInputChannelName).at(0);
+        fair::mq::Channel& inputChannel = fairroot::GetFairMQDeviceChannels(*this).at(fInputChannelName).at(0);
```

Comment:
Would a `auto&` work here? Would it be a good idea? Or is it better to keep the type explicit?

Reply:
done (and changed it to the `GetChannel` api)